src/darwin/Framework/CHIP/CHIPDeviceController.mm

```diff
@@ -182,6 +190,7 @@ - (BOOL)shutdown
 - (BOOL)startup:(_Nullable id<CHIPPersistentStorageDelegate>)storageDelegate
        vendorId:(uint16_t)vendorId
       nocSigner:(id<CHIPKeypair>)nocSigner
+       paaCerts:(nullable NSArray<NSData *> *)paaCerts
 {
     return [self startup:storageDelegate vendorId:vendorId nocSigner:nocSigner ipk:nil];
```

Comment:
Need to add `paaCerts` to the other `startup` signature too, and pass it through from here.  This is why there are compilation errors.

Reply:
Done